<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 118, 123);
border-color: rgb(119, 118, 123);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lab_current_shceme">
         <property name="text">
          <string>当前方案：XXXXXXXXXXX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_scheme">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_load_shceme">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>加载</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_edit_shceme">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>编辑</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lab_current_device">
         <property name="text">
          <string>当前设备：XXXXXXXXXXX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_switch_device">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>切换设备</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="contentWidget" native="true">
      <layout class="QHBoxLayout" name="contentHLayout" stretch="1">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QWidget" name="videoWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="toolWidget" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btn_show_crossing_line">
               <property name="text">
                <string>十字</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_shrink">
               <property name="text">
                <string>缩小</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_enlarg">
               <property name="text">
                <string>放大</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_original">
               <property name="text">
                <string>还原</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lab_display">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
